test/parse/expr/bad-simd-shuffle-nat-expected.txt

```diff
@@ -9,7 +9,7 @@
     ))
 
 (;; STDERR ;;;
-out/test/parse/expr/bad-simd-shuffle-nat-expected.txt:8:19: error: unexpected token "$0x01020304", expected an Nat literal (e.g. 123).
+out/test/parse/expr/bad-simd-shuffle-nat-expected.txt:8:19: error: unexpected token "$0x01020304", expected a natural number (e.g. 123).
```

Comment:
This error will be a bit confusing, because while `123` is a natural number, it will be out-of-bounds for the `shuffle` instruction.